Core/Types/Version.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Text.RegularExpressions;
 using Newtonsoft.Json;
+using System.Reflection;
```

Comment:
Experimented with another method and forgot to install Resharper on this machine, so VS didn't auto-remove and sort usings.